graphql_compiler/cost_estimation/statistics.py

```diff
@@ -24,29 +24,30 @@ def __repr__(self):
 
     @abstractmethod
     def get_class_count(self, class_name):
-        """Return how many objects have, or inherit, the given class name.
+        """Return how many vertex or edge instances have, or inherit, the given class name.
 
         Args:
             class_name: str, either a vertex class name or an edge class name defined in the
                         GraphQL schema.
 
         Returns:
-            - int, number of objects with, or inheriting the given class name
+            - int, number of vertices and edges with, or inheriting the given class name
```

Comment:
nit: Either remove instances from the docstring above or add "instances" after vertices and edges.